### DISCUSSION TITLE
Connecting to a postgres db in Dbeaver through a jupyter notebook

### DISCUSSION THREAD

**Discussion by singhv007** [dbeaver/dbeaver#35077] [General]
I believe this must be a solved problem -- I am trying to connect to a Postgres db through my DBeaver account and run queries directly from my Jupyter Notebook. Hoping someone can point me to the solution

**Comment by TB-HunBoka** [comment]
Hello @singhv007 ,

I think I can't answer your question, but to be honest, I am not sure what the goal is you would like to achieve. Could you please explain in more detail why do you need your DBeaver "account" to run query in Jupyter Notebook?
And what do you mean by DBeaver Account, did you buy the pro version? Or what is a DBeaver Account?
You can connect to postgresql from jupyter lab / jupyter notebook with SQL alchemy for example, if that helps.
